server/config/runtime.exs

```diff
@@ -330,14 +330,13 @@ config :tuist, Oban,
     {Oban.Plugins.Lifeline, rescue_after: to_timeout(minute: 30)},
     {Oban.Plugins.Cron,
      crontab:
-       if(Tuist.Environment.tuist_hosted?() and env == :prod,
+       if(Tuist.Environment.tuist_hosted?(),
          do: [
            {"0 10 * * 1-5", Tuist.Ops.DailySlackReportWorker},
            {"0 * * * 1-5", Tuist.Ops.HourlySlackReportWorker},
            {"@hourly", Tuist.Registry.Swift.Workers.SyncPackagesWorker},
```

Comment:
This means we'll sync the whole Swift Package Index also in staging and canary. This may or may not be what we want. We can reconsider if it means we'd need higher specs for these envs, but otherwise, this might be fine.